frontend/src/components/common/LogComponent.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/react-in-jsx-scope */
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Select,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+
+type LogComponentProps = {
+  employee: string;
+};
+
+// eslint-disable-next-line react/prop-types
+const LogComponent: React.FC<LogComponentProps> = ({ employee }) => {
+  const [building, setBuilding] = useState<string>("");
+  const [tags, setTags] = useState<string>("");
+
+  const handleBuildingChange = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ) => {
+    setBuilding(event.target.value);
+  };
+
+  const handleTagsChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTags(event.target.value);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    // eslint-disable-next-line no-console
+    console.log(
+      `Employee: ${employee}, Date: ${new Date()}, Building: ${building}, Tags: ${tags}`,
+    );
+  };
+
+  return (
+    <div className="log-entry">
+      <h1>New Log Entry Details</h1>
+      <form onSubmit={handleSubmit}>
+        <FormControl>
+          <FormLabel>Employee</FormLabel>
```

Comment:
For required fields, there should be a red asterisk next to the label like so:
![image](https://user-images.githubusercontent.com/57375371/230035254-b9fb5db4-8768-4832-b2b4-8ee0e8c6e7b6.png)

I believe you can do this in chakra by passing the `isRequired` prop to the FormControl component:
```
            <FormControl
              id="first-name"
              className="body"
              marginTop="16px"
              isRequired
            >
            ...
            </FormControl>
```